cypress/support/commands/analytics.js

```diff
@@ -15,12 +15,14 @@ Cypress.Commands.add('hasNoscriptImgAtiUrl', atiUrl => {
 
 // Should be moved into integration/pages/index.js once all pages have Chartbeat
 Cypress.Commands.add('hasScriptWithChartbeatSrc', () => {
+  cy.reload(true);
```

Comment:
Don't particularly like this, or the `{ retries: 3 }` on the individual Chartbeat tests, but I've tried everything to get them to work and couldn't find a better solution.

Feel like it _may_ have something to do with how React 19 takes over `<script />` tags, but can't be sure.